changelog.adoc

```diff
@@ -9,16 +9,24 @@ and this project tries to adhere to https://semver.org/spec/v2.0.0.html[Semantic
 
 === fixes
 
-* fix decision tree for `dtcw.ps1`
-* #831 : fix for generateSite: images-hrefs not correct
-* #1109 : docToolchain release notes contain releases twice
+* https://github.com/docToolchain/docToolchain/issues/621[#621: dtcw - sdkman installation check returns wrong result]
+* https://github.com/docToolchain/docToolchain/issues/831[#831: fix for generateSite: images-hrefs not correct]
+* https://github.com/docToolchain/docToolchain/issues/973[#973: `dtcw getJava` doesn't work without `local`]
+* https://github.com/docToolchain/docToolchain/issues/1109[#1109: docToolchain release notes contain releases twice]
+* https://github.com/docToolchain/docToolchain/issues/1031[#1031: dtcw ignores installed Java RE when docker is installed - your java version 17 is too new]
+* various fixes in `dtcw`, `dtcw.ps1`:
+** fix decision tree for `dtcw.ps1`
```

Comment:
@rdmueller What decision tree did you mean here? I can't understand what you mean.

Reply:
Ahh, OK. I try to describe this in more details.